db/migrate/20230920093000_move_ct_command_into_own_setting.rb

```diff
@@ -0,0 +1,24 @@
+class MoveCtCommandIntoOwnSetting < ActiveRecord::Migration[6.1]
+  def up
+    ['ct', 'fcct'].each do |setting_prefix|
+      setting = Setting.where(name: "#{setting_prefix}_command")
+      if setting
+        setting.value = setting.value[1..]
+        setting.name = "#{setting_prefix}_arguments"
+        setting.save
+      end
+    end
+  end
+
+  def down
```

Comment:
not sure how much benefit there is for the `down` method here

Reply:
Often overlooked, it's useful sometimes when debugging, you already wrote it so keep it.